makeflow/src/makeflow.c

```diff
@@ -2046,14 +2078,13 @@ if (enforcer && wrapper_umbrella) {
 
 	makeflow_log_started_event(d);
 
-
 	runtime = timestamp_get();
 
 	if (container_mode == CONTAINER_MODE_DOCKER) {
-            makeflow_wrapper_docker_init(wrapper, container_image, image_tar);
+    	makeflow_wrapper_docker_init(wrapper, container_image, image_tar);
 	}else if(container_mode == CONTAINER_MODE_SINGULARITY){
-            makeflow_wrapper_singularity_init(wrapper, container_image);
-        }
+    	makeflow_wrapper_singularity_init(wrapper, container_image);
```

Comment:
@nhazekam What tool are we using for dealing with tab and whitespace? I have been bothered by this problem by a long time.